code/src/index.css

```diff
@@ -1,13 +1,19 @@
+* {
+  box-sizing: border-box;
```

Comment:
good practice is to use the following for border box 😄 
`*, *::after, *::before { box-sizing:border-box; }`

Reply:
👍